### DISCUSSION TITLE
Unable to access Postal backend

### DISCUSSION THREAD

**Discussion by anvishinc** [postalserver/postal#2204] [Help with using Postal]
So, I followed all the step as mentioned in the documentation while installing it on Ubuntu 22.04 arm64 over on oracle cloud. I also allowed ports 80, 5000, 25, and the rest. pointed my DNS record and everything. But no matter what I am not able to access the backend. I keep getting connection refused.

**Comment by willpower232** [comment]
I think if you try using `postal status` you'll find that your Postal did not start and this is probably because I don't think its an ARM-compatible image

**Reply by anvishinc** [reply]
Did check all of those multiple times, everything was working fine. That is why I am having a hard time figuring out what is going wrong. Also, I tried with Oracle cloud AMD server running Ubuntu 22.04 and had the same issue.

**Reply by willpower232** [reply]
Is your nginx or caddy working?

**Reply by anvishinc** [reply]
yup both are working fine I was able to access a test page for nginx and caddy was running without any issues

**Reply by anvishinc** [reply]
based on the lack of documentation for OCI I am starting to think that might be the problem here. But I just am not able to figure out what in OCI might be causing this issue

**Reply by willpower232** [reply]
well technically you only need one of nginx and caddy, running both is likely to get confusing

I don't know anything about oracle but if you've opened the firewall on the server, is there another firewall within the oracle settings directly?

**Comment by kluvi** [comment]
Maybe problem with firewall? Some combinations with older docker daemon+firewalld doesnt work by-default (docker rules were not applied, so containers have broken network).

You can try to get inside some of the broken containers and try to ping for example `google.com`
